source/includes/steps-create-index.yaml

```diff
@@ -27,18 +27,55 @@ content: |
 
    |compass-short| supports the following index options:
 
-   * `Build the index in the background <https://docs.mongodb.com/master/core/index-creation/index.html#background-construction>`_
+   .. list-table::
+      :header-rows: 1
+      :widths: 40 60 20
 
-   * :manual:`Create a unique index </core/index-unique>`
+      * - Option
+        - Description
+        - More Information
 
-   * :manual:`Create a TTL Index </core/index-ttl>`
+      * - Build index in the background
 
-   * :manual:`Create a partial index </core/index-partial/>`
+        - If checked, the MongoDB deployment remains available during
```

Comment:
This description isn't quite parallel with the others.

Reply:
Agreed. Tweaked a bit to try to make it a bit more parallel.